File: src/main/java/com/graqr/threshr/model/redsky/store/Capability.java
package com.graqr.threshr.model.redsky.store;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Capability(@JsonProperty("capability_code") String capabilityCode,
                         @JsonProperty("capability_name") String capabilityName,
                         @JsonProperty("effective_date") String effectiveDate) {}

File: src/main/kotlin/com/graqr/threshr/model/redsky/store/Capability.kt
package com.graqr.threshr.model.redsky.store

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Capability(
    @param:JsonProperty("capability_code") val capabilityCode: String,
    @param:JsonProperty("capability_name") val capabilityName: String,
    @param:JsonProperty("effective_date") val effectiveDate: String
) {}
